sound/soc/intel/boards/Kconfig

```diff
@@ -510,7 +511,7 @@ config SND_SOC_INTEL_SOF_CML_RT1011_RT5682_MACH
 	tristate "CML with RT1011 and RT5682 in I2S Mode"
 	depends on I2C && ACPI
 	depends on MFD_INTEL_LPSS || COMPILE_TEST
-	depends on SND_HDA_CODEC_HDMI
+	depends on SND_HDA_CODEC_HDMI && SND_SOC_SOF_HDA_AUDIO_CODEC
```

Comment:
same as above

Reply:
@kv2019i Do you mean the same as "https://github.com/thesofproject/linux/pull/1989#discussion_r404732783". In this config, I would prefer "depends on SND_HDA_CODEC_HDMI && SND_SOC_SOF_HDA_AUDIO_CODEC" to “depends on SND_SOC_SOF_HDA_AUDIO_CODEC && (MFD_INTEL_LPSS || COMPILE_TEST)”. This is because I need "SND_SOC_SOF_HDA_AUDIO_CODEC" is enabled only because we are using SND_HDA_CODEC_HDMI. So it's better we put them together

Reply:
@libinyang Fair enough, for I2S machine drivers this makes sense indeed.